models/chainlink/ethereum/chainlink_ethereum_CCIP_schema.yml

```diff
@@ -0,0 +1,75 @@
+version: 2
+
+models:
+  - name: chainlink_ethereum_ccip_onramps
+    meta:
+      alias: chainlink_ethereum_ccip_onramps_ETH
```

Comment:
You can remove all alias fields under meta maybe.

Reply:
First time making a schema so sorry if things seem a little weird. But there was an issue with running dbt compile that seemed to go away when making them into meta alias. I'll remove them and check it locally & push if it seems to work

Reply:
No worries, thank you for your contribution.